### PR TITLE
Fix gallery occasionally displaying incorrect images

### PR DESCRIPTION
**Pull request by alan-wu**
# Description

This addresss the issue reported here - https://www.wrike.com/open.htm?id=1059863316
The solution is to use datacite from scicrunch to look for the correct thumbnails.
Include a minor fix which causes yarn dev to fail to load the page.

## Type of change

Delete those that don't apply.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

The current site displays incorrect thumbnail with this dataset - https://sparc.science/datasets/34?type=dataset&datasetDetailsTab=images

A site with the deployed fix here - https://alan-wu-sparc-app.herokuapp.com/datasets/34?type=dataset&datasetDetailsTab=images

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have utilized components from the Design System Library where applicable
- [ ] I have added unit tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: components/ImagesGallery/ImagesGallery.vue [hunk 1]
```diff
@@ -445,6 +448,21 @@ export default {
             ...Array.from(scicrunchData['abi-plot'], plot => {
               const id = plot.identifier
               const file_path = plot.dataset.path
+              const thumbnail = this.getThumbnailForScaffold(
```
Comment (Tehsurfer) [inline components/ImagesGallery/ImagesGallery.vue:451]: Maybe this function could be named `getThumbnailForScicrunchItem`? 

And then `getThumbnailsForPlot` could be deleted.

**Review by Tehsurfer — APPROVED** [review]
Works well! 

there are a few confusing named functions in this file it seems 

1.  `getThumbnailForScaffold` being used on plots as well
2.  ` getThumbnailForScaffold` probably should be `getThumbnailPathForScaffold
3. `getThumbnailWithInfo` probably should be `retrieveThumbnailFromInfo`

But those aren't from this PR, just something I noticed

File: components/ImagesGallery/ImagesGallery.vue (hunk 1 above)
Comment (alan-wu) [reply components/ImagesGallery/ImagesGallery.vue:451]: Good catch, they work differently and plot items should use the one for plot instead.

Comment (alan-wu) [reply components/ImagesGallery/ImagesGallery.vue:451]: I like those method names.

**Pull request merged**